src/Elastic.Apm/Model/Payload/Context.cs

```diff
@@ -5,11 +5,12 @@ namespace Elastic.Apm.Model.Payload
 {
 	internal class Context
 	{
-		private readonly Lazy<Dictionary<string, string>> tags = new Lazy<Dictionary<string, string>>();
-		public Request Request { get; set; }
+		private readonly Lazy<Dictionary<string, string>> _tags = new Lazy<Dictionary<string, string>>();
+		public Lazy<User> User { get; set; }
+		public Lazy<Request> Request { get; set; }

```

Comment:
These remain `null`. You'll need a field and initialize that with `Lazy` and you need to return that. Like this: 
```
private readonly Lazy<Request> _request = new Lazy<Request>();
public Request Request => _request.Value;
```

Same for the other properties.

Reply:
This way you'll also not need to change the user code... so the property itself will have the same type as before (e.g. `Request` and not `Lazy<Request>`), just the backing field will be `Lazy`.

Reply:
fixed thanks @gregkalapos